src/libs/utils.js

```diff
@@ -254,3 +254,8 @@ export const withBusyState = _.curry((setBusy, fn) => async (...args) => {
     setBusy(false)
   }
 })
+
+// Performs the given effect, but only on component mount
+export const useOnMount = fn => {
```

Comment:
I called this one `useOnMountOnly` to be even more explicit, either way is cool tho